packages/teams-js/src/public/runtime.ts

```diff
@@ -66,14 +98,14 @@ export interface IRuntime {
  * @internal
  * Limited to Microsoft-internal use
  */
-export const _uninitializedRuntime = {
-  apiVersion: -1,
+export const _uninitializedRuntime: Runtime = {
+  apiVersion: 1,
```

Comment:
Out of curiosity, what caused you to need to bump this from -1 to 1? Since it's uninitialized I assumed it should be set to some sort of error value

Reply:
The version needs to be 1 to conform to the Runtime interface so it can be assigned to `runtime`. I looked into making `runtime` a union type with `UnitializedRuntime`, but we would have to type check `runtime` everywhere we use it. I like the idea of having an error value here, but we weren't actually checking for it anywhere, right? Maybe something like an `isInitialized`/`isUnintialized` flag would be helpful?

Reply:
Yes, I don't think we were checking it anywhere it was mostly just curiosity. Does that uninitializedRuntime object actually match what we expect from version 1? It looks like it doesn't have the expected capabilities in the supports object

Reply:
Updated to use an `IUninitializedRuntime` interface for the uninitialized runtime